lib/purescript-cst/src/Language/PureScript/CST/Errors.hs

```diff
@@ -128,6 +129,10 @@ prettyPrintErrorMessage (ParserErrorInfo {..}) = case errType of
     "Illegal whitespace character " <> displayCodePoint hd
   ErrLexeme (Just a) _ ->
     "Unexpected " <> a
+  ErrQualifierLexeme hd | isSpace hd ->
+    "Unexpected whitespace character " <> displayCodePoint hd <> ", expected qualifier"
```

Comment:
Don't you mean ‘expected identifier or operator’ instead of ‘expected qualifier’? The qualifier has already been seen at this point in the parse, right?

Also, it looks like the other error messages of this nature follow an ‘Expected X, saw Y’ pattern; I'd expect these messages to match that.